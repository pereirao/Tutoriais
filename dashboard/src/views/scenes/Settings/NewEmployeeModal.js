// @flow

import React from 'react';
import NewEmployeeForm from './NewEmployeeForm';
import style_sheet from './Settings.module.css';
import I18n from '../../../localization';
import MBModal from '../../elements/MBModal';

type NewEmployeeModalProps = {
  hidden: boolean,
  close: () => void
};

const NewEmployeeModal = ({hidden, close}: NewEmployeeModalProps) => {
  if (hidden) return null;
  return (
    <MBModal close={close}>
      <h3 className={style_sheet.modalHeader}>{I18n.t('ui.modal.new_employee.title')}</h3>
      <NewEmployeeForm close={close}/>
    </MBModal>
  );
};

export default NewEmployeeModal;
