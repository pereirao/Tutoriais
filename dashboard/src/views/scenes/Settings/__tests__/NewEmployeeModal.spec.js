import React from 'react';
import NewEmployeeModal from '../NewEmployeeModal';

it('renders correctly', () => {
  expect(shallow(<NewEmployeeModal />)).toMatchSnapshot();
});
