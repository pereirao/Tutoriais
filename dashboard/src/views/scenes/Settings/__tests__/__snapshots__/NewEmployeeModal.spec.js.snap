// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<MBModal
  closeable={true}
  size="medium"
>
  <h3
    className="modalHeader"
  >
    Add a New Employee
  </h3>
  <Connect(NewEmployeeForm) />
</MBModal>
`;
